Mathlib/Algebra/BigOperators/Group/List.lean

```diff
@@ -542,6 +565,24 @@ theorem prod_set' (L : List G) (n : ℕ) (a : G) :
 #align list.prod_update_nth' List.prod_set'
 #align list.sum_update_nth' List.sum_set'
 
+@[to_additive]
+lemma prod_map_ite_eq {A : Type} [DecidableEq A] (l : List A) (f g : A → G) (a : A) :
+    (l.map (fun x => ite (x = a) (f x) (g x))).prod
+      =
+    (f a / g a) ^ (l.count a) * (l.map g).prod := by
+  induction l with
+  | nil => simp
+  | cons x xs ih =>
+    simp only [map_cons, prod_cons, nodup_cons, ne_eq, mem_cons, count_cons] at ih ⊢
+    rw [ih]
+    clear ih
+    by_cases hx : x = a
+    · simp only [hx, ↓reduceIte, div_pow, pow_add, pow_one]
+      -- TODO replace with `abel`
```

Comment:
Please clarify. Why haven't you? If it doesn't work, is there a reason to assume it will work in the future?

Reply:
Actually, seems `abel` itself is not supposed to be imported in this file at all, so I'll just remove the comment.

Reply:
(and use the golf technique from [here](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Dependencies.20of.20abel))